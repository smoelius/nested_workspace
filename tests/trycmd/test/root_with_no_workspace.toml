args = ["test", "--workspace"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/root_with_no_workspace"
