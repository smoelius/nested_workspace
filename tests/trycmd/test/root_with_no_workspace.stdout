
running 1 test
test [PUT] ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [..]s

